Dockerfile

```diff
@@ -10,7 +10,7 @@ ADD . /opt/crystal-head
 
 WORKDIR /opt/crystal-head
 ENV CRYSTAL_CONFIG_VERSION HEAD
-ENV CRYSTAL_CONFIG_PATH lib:libs:/opt/crystal-head/src:/opt/crystal-head/libs
+ENV CRYSTAL_CONFIG_PATH lib:/opt/crystal-head/src:/opt/crystal-head/libs
```

Comment:
@luislavena `CRYSTAL_CONFIG_PATH` is only used for `Crystall::Config.path`.
I've removed the complete line `CRYSTAL_CONFIG_PATH` and `docker build .` still produced a valid crystal compiler.

I think we can remove the whole line.